User.py

```diff
@@ -95,3 +95,14 @@ def addExercise(self, exercise, reps):
     def hasDoneExercise(self, exercise):
         return exercise["id"] in self.exercise_counts
 
+
+def parseSlackJSON(response):
+    isMessageOkay = False
+    try:
+        if response.ok:
+            parsed_message = json.loads(response.text, encoding='utf-8')
+    except:
+        print "Caught exception parsing response status: " + str(response.status_code) + ", text: " + response.text
+
+    isMessageOkay = response.ok & parsed_message["ok"]
```

Comment:
Then use `and`. I _think_ it'll work fine with the bitwise operator, but we're looking to return strictly `True` or `False`